src/qt/addressbookpage.cpp

```diff
@@ -293,9 +293,11 @@ void AddressBookPage::done(int retval)
 void AddressBookPage::on_exportButton_clicked()
 {
     // CSV is currently the only supported format
-    QString filename = GUIUtil::getSaveFileName(this,
+    QString filename = GUIUtil::getSaveFileName(
+        this,
         tr("Export Address List"), QString(),
-        tr("Comma separated file (*.csv)"), nullptr);
+        // TODO: Replace QStringLiteral with QLatin1String when Qt minimum version >= 5.10.
+        tr("Comma separated file (%1)").arg(QStringLiteral("*.csv")), nullptr);
```

Comment:
IIRC the parenthesis format is also special, to trigger the dialog filtering files shown. So perhaps this should be:
```suggestion
        tr("Comma separated file") + QStringLiteral(" (*.csv)"), nullptr);
```

Reply:
Thank you! [Updated](https://github.com/bitcoin-core/gui/pull/220#issuecomment-785364090).